features/networking/operator.feature

```diff
@@ -13,3 +13,167 @@ Feature: Operator related networking scenarios
     Then the expression should be true> cb.operator_status["status"]=="True"
     #Confirm whether network operator version matches with ocp version
     And the expression should be true> cluster_operator('network').version_exists?(version: cb.ocp_version)
+
+
+  # @author bmeng@redhat.com
+  # @case_id OCP-22201
+  @admin
+  Scenario: Should have a clusteroperator object created under config.openshift.io api group for network-operator
+    Given the master version > "3.11"
```

Comment:
Will this work on 3.12 if it is ever released? I doubt we will ever have such version but in any case, better do as
```
    Given the master version >= "4.0"
```